airflow-core/docs/core-concepts/executor/local.rst

```diff
@@ -21,32 +21,29 @@
 Local Executor
 ==============
 
-:class:`~airflow.executors.local_executor.LocalExecutor` runs tasks by spawning processes in a controlled fashion in different modes.
+:class:`~airflow.executors.local_executor.LocalExecutor` runs tasks by spawning processes in a controlled fashion on the scheduler node.
 
-Given that BaseExecutor has the option to receive a ``parallelism`` parameter to limit the number of process spawned,
-when this parameter is ``0`` the number of processes that LocalExecutor can spawn is unlimited.
+The parameter ``parallelism`` limits the number of process spawned not to overwhelm the node.
+This parameter must be greater than ``0``.
```

Comment:
Base executor init will still allow parallelism to be equal to zero. Which I suppose you may want, to disable an executor from taking tasks? Not sure I really think that's a valid use case but the code does allow it, so we should either change that or update this:


```suggestion
This parameter must be greater than or equal to ``0``.
```